### PR TITLE
Implement tests for amity space allocation

### PR DESCRIPTION
**Pull request by gacheruevans**
##What does this PR do?
Run Amity Allocation System for Testing.

##Description of task to be completed.
Create Amity Allocation CLI Python System that:

creates Fellows and Staff
creates Living Spaces and Offices
relocate Fellow/Staff from one office to another
get allocations and unallocated person from the system

##How should this be manually tested?
python run.py -i

### CODE REVIEW

File: app/rooms/roomClass.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        living_space_allocations = []
+        people_without_living_spaces = []
+
+        for room_name, room_info in rooms.items():
+            if room_info["is_office"]:
+                office_allocations += room_info["occupants"]
+            if not room_info["is_office"]:
+                living_space_allocations += room_info["occupants"]
+
+        unllocated_offices = list(
```
Comment (arnoldokoth) [inline app/rooms/roomClass.py]: Slight typo here `unllocated_offices` should probably be `unallocated_offices`

Comment (gacheruevans) [reply app/rooms/roomClass.py]: True will amend that thank you for pointing it out.

File: .coverage [hunk 2]
```diff
@@ -1 +1 @@
...
.../dist-packages/pbr/version.py": [19, 21, 22, 23, 25, 28, 29, 30, 31, 36, 40, 43, 44, 56, 57, 58, 59, 60, 61, 63, 65, 70, 73, 96, 106, 109, 112, 115, 118, 121, 146, 147, 151, 154, 156, 158, 159, 160, 162, 174, 175, 176, 177, 178, 179, 180, 182, 190, 191, 197, 198, 199, 200, 204, 208, 213, 225, 226, 227, 228, 234, 236, 238, 240, 248, 277, 319, 327, 332, 333, 337, 344, 346, 351, 353, 363, 372, 388, 389, 400, 404, 405, 406, 409, 411, 417, 418, 419, 420, 422, 426, 431, 438, 439, 440, 441, 448, 450, 457, 459, 460, 461, 463, 468, 469, 471], "/usr/local/lib/python2.7/dist-packages/funcsigs/__init__.py": [6, 7, 8, 9, 10, 11, 13, 14, 656, 18, 451, 20, 277, 23, 24, 281, 26, 27, 28, 285, 31, 289, 411, 293, 294, 39, 801, 794, 455, 626, 486, 659, 596, 446, 55, 442, 315, 190, 191, 339, 194, 195, 198, 199, 200, 201, 202, 343, 204, 207, 211, 212, 213, 214, 215, 787, 603, 218, 347, 476, 478, 607, 480, 481, 354, 483, 485, 358, 236, 238, 240, 241, 242, 243, 244, 373, 246, 375, 248, 249, 379, 523, 622, 383], "/home/andela/andela/andelasims/cp1-amity/tests/__init__.py": [1], "/usr/local/lib/python2.7/dist-packages/funcsigs/version.py": [1], "/usr/local/lib/python2.7/dist-packages/pbr/__init__.py": [1]}}
```
Comment (andela-landia) [inline .coverage]: This file should not be committed add it  to your gitignore file and remove it from git

**Review by andela-landia — COMMENTED** [review]
Do not commit generated files such as pyc and .coverage

File: app/database/database.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
+import os
+from app.database.models import *
+from app.amity import my_amity
```
Comment (andela-landia) [inline app/database/database.py:3]: Arrange imports alphabetically

File: app/rooms/roomClass.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            set(Person.people_data.keys()) - set(office_allocations))
```
Comment (andela-landia) [inline app/rooms/roomClass.py]: You could check out set symmetric difference

File: app/rooms/test_room.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import unittest
+import unittest.mock as mock
```
Comment (andela-landia) [inline app/rooms/test_room.py]: Separate imports of files you have created from python libraries such as mock

File: requirements.txt [hunk 6]
```diff
@@ -1,4 +1,28 @@
...
+ipdb==0.10.1
+ipython==5.1.0
+ipython-genutils==0.1.0
+mock==2.0.0
 nose==1.3.7
+pbr==1.10.0
+pexpect==4.2.1
+pickleshare==0.7.4
+prompt-toolkit==1.0.9
+ptyprocess==0.5.1
```
Comment (andela-landia) [inline requirements.txt:16]: Clean up your requirements file removing subsidiary libraries that are installed with outher libraries

File: app/database/database.py (hunk 3 above)
Comment (gacheruevans) [reply app/database/database.py:3]: will work on that, thank you for pointing it out.

File: app/rooms/roomClass.py (hunk 4 above)
Comment (gacheruevans) [reply app/rooms/roomClass.py]: thank you for highlighting that, will certainly read up on 'symmetric difference'

File: requirements.txt (hunk 6 above)
Comment (gacheruevans) [reply requirements.txt:16]: will do amend that, thank you for pointing that out.

**Pull request merged**